app/User.php

```diff
@@ -47,37 +48,24 @@ class User extends Authenticatable
     public function storeDevice(LoginRequest $request)
     {
        $device = Device::updateOrCreate([
-        'device_token' => $request->device_token, 
-        'device_type' => $request->device_type,
-        'user_id' => $request->user_id,
-        ],[
-        'api_token' =>str_random(60)
-        ]);
+                'device_token' => $request->device_token, 
+                'device_type'  => $request->device_type,
+                'user_id'      => $request->user_id,
+                ],[
```

Comment:
Assign two variables initially, and pass the variable inside the `updateOrCreate` method